rdkit/Chem/Draw/__init__.py

```diff
@@ -427,6 +428,230 @@ def ReactionToImage(rxn, subImgSize=(200,200),**kwargs):
     res.paste(nimg,(i*subImgSize[0],0))
   return res
 
+def is2D(conf):
+  """returns True if a conf has 2d coords"""
+  allZeroes = True
+  for atompos in range(conf.GetNumAtoms()):
+    x,y,z = conf.GetAtomPosition(atompos)
+    if z != 0.0:
+      return False
+    if x != 0.0 or y != 0.0:
+      allZeroes = False
+  return not allZeroes
+
+def _cleanupRXN(rxn):
+  """Cleanup reactions so the computed coords are sane for depiction"""
+  for mol in rxn.GetReactants():
+    compute2D=False
+    for conf in mol.GetConformers():
+      compute2D = not is2D(conf)
+
+    try:
+      AllChem.SanitizeMol(mol, 
+                          sanitizeOps=(AllChem.SanitizeFlags.SANITIZE_ALL^ 
+                                       AllChem.SanitizeFlags.SANITIZE_KEKULIZE))
+    except:
+      pass
+    if compute2D:
+      AllChem.Compute2DCoords(mol)
+
+  for mol in rxn.GetProducts():
+    compute2D=False
+    for conf in mol.GetConformers():
+      compute2D = not is2D(conf)
+    try:
+      AllChem.SanitizeMol(mol, 
+                          sanitizeOps=(AllChem.SanitizeFlags.SANITIZE_ALL^ 
+                                       AllChem.SanitizeFlags.SANITIZE_KEKULIZE))
+    except:
+      pass
+    if compute2D:
+      AllChem.Compute2DCoords(mol)  
+  return rxn
+
+#source https://commons.wikimedia.org/wiki/File:Tab_plus.svg
+#  creative commons license (need attribution?  Is link enough? Make our own?)
```

Comment:
Given the simplicity (and I guess it could be a lot simpler) and the fact that the wikimedia object is "Share alike" licensed we may want to re-do this. I believe that this would count as a re-distribution and the RDKit's BSD license is not the same as CC-BYSA, so it's probably not technically correct to re-use it.